packages/v4/src/content/design-guidelines/components/page/page.md

```diff
@@ -2,4 +2,58 @@
+
+**3. Breadcrumb (optional):** When breadcrumbs are included, they should be the topmost element in the header section. When breadcrumbs are included on a page, the top padding is reduced to 16px. See the [Breadcrumb design guidelines](/components/breadcrumb/design-guidelines) for more information about breadcrumb usage.
+
+**4. Page header:** All pages should have a header section. The height of the header will vary to fit the content. We recommend keeping information in the header area brief to consume as little space as possible. The page header should contain the page title to help orient the user and an optional description to describe the contents of the page. You may optionally apply the `pf-m-sticky-top` modifier to the header section to create a sticky page header and prevent its content from scrolling off the page.
+
+**5. Body section:** Your main page content goes here. The body section can have a light gray `--pf-global--BackgroundColor--200` or white background depending on the content it will contain. The default padding all around the page body should be 24px in all directions.
+
+**6. Footer section (optional):** You can optionally define a footer section for a page. You may optionally apply the `pf-m-sticky-bottom` modifier to the footer section to create a sticky page footer and prevent its content from scrolling off the page.
+
+### Page with horizontal navigation
+
+<img src="./img/page-horizontal-nav.png" alt="page with horizontal navigation" width="832"/>
+
+When using a horizontal primary navigation, the left-hand sidebar is removed and the navigation is placed inline with other masthead elements. All other page elements remain the same.
+
+## Masthead
+The masthead contains global properties such as logotype, navigation and settings in an organized fashion and it is accessible from all parts of the application. All pages should share a common masthead.
+
+<img src="./img/masthead.png" alt="masthead" width="832"/>
+
+**1. Show/hide menu:**  provides a way for users to toggle vertical navigation. We recommend always having the menu icon visible when using vertical navigation. Show/hide menu is also used when application is being displayed in smaller viewport sizes, and it should stay visible in smaller viewport sizes.
+
+**2. Logotype:** The logotype area should clearly display the application name and the application logo if applicable. It is common to link the logotype to the application’s homepage. It is recommended to have the application name in .svg format for better performance.
+
+**3. Utility items:** Place any items in the utility area that you want to have accessible from all pages in the application. Utility items are right aligned after the user menu. The following are common items ordered from left to right. All items are optional depending on the needs of your application.
```

Comment:
The sentence "Utility items are right aligned after the user menu" took me a second to understand. Would maybe change to something like "Utility items are right alighted on the masthead, and placed to the left of the user menu"

Reply:
Good point. Changed to "Utility items are right aligned and placed to the left of the user menu."